File: Math_3/5086.py
def main(a, b):
    if a%b==0:
        print("multiple")
    elif b%a==0:
        print("factor")
    else:
        print("neither")    


if __name__ == "__main__":
    while True:
        a, b = input().split()
        a = int(a); b = int(b)
        if a==0 and b==0:
            break
        else:
            main(a,b)

File: Math_3/5086.java
package pr;

import java.io.BufferedReader;
import java.io.InputStreamReader;

import java.util.StringTokenizer;

public class Main {
	public static void main(String[] args) throws Exception {
	BufferedReader bf=new BufferedReader(new InputStreamReader(System.in));
	StringTokenizer st=new StringTokenizer(bf.readLine());

	
	int x=Integer.parseInt(st.nextToken());
	int y=Integer.parseInt(st.nextToken());
	
	while(x!=0 && y!=0) {
		if(y%x==0) System.out.println("factor");
		else if(x%y==0) System.out.println("multiple");
		else System.out.println("neither");
		st=new StringTokenizer(bf.readLine());
		x=Integer.parseInt(st.nextToken());
		y=Integer.parseInt(st.nextToken());
		
	}
	
	
	}
}




